### PR TITLE
fix: UI Misalignment and Broken Layout on User Application

### LINKED ISSUE #912: UI Misalignment and Broken Layout on User Application
### Issue Description
There is a critical UI misalignment and broken layout observed in the user application interface. This issue affects user navigation and usability, creating a frustrating experience and impairing core functionalities.

### Screenshots
image
image

### Reproducibility

 - [ ] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. 
2. 

### Severity/Priority
 - [ ] Critical
 - [x] High
 - [ ] Medium
 - [ ] Low

### Checklist

- [ ] I have read and followed the project's code of conduct.
- [ ] I have searched for similar issues before creating this one.
- [ ] I have provided all the necessary information to understand and reproduce the issue.
- [ ] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by vamsi4845**
Date: 

Developer Name: Vamsi Karumuri

---

## Issue Ticket Number

- Closes #912 

## Description

This PR resolves  issue related to application cards content ui misalignment on applications page
### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

https://github.com/user-attachments/assets/ec2cb053-8098-4f5f-89c4-53ab2f369f61

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

Photos_eZRbQmRL4p

</details>

### CODE REVIEW

**Comment by AnujChhikara** [comment]
Why this is not fix?

**Comment by vamsi4845** [comment]
> Why this is not fix?
> 
>  

ok fixed this,Check once and confirm 

https://github.com/user-attachments/assets/076e4a2b-f63f-467d-9d19-3c57b3195835

**Comment by vamsi4845** [comment]
@AnujChhikara I've updated the code, Please review it

**Review by AnujChhikara — APPROVED** [review]

File: global.css [hunk 1]
```diff
@@ -176,7 +176,6 @@ body {
 .footer {
   width: 100vw;
   text-align: center;
-  position: fixed;
```
Comment (AnujChhikara) [inline global.css:179]: did you check if removing this position:fixed don't impact other pages

Comment (vamsi4845) [reply global.css:179]: Yes I've checked

**Comment by vamsi4845** [comment]
@AnujChhikara @vinit717 Can you review the PR?

**Comment by AnujChhikara** [comment]
> @AnujChhikara @vinit717 Can you review the PR?

Are you in the Discord server? If so, could you please post in the group to request a review?

File: applications/style.css [hunk 2]
```diff
@@ -350,6 +352,7 @@ body {
   font-size: 16px;
   padding: 6px 16px;
   margin-left: auto;
+  margin-top: 5%;
```
Comment (vinit717) [inline applications/style.css:355]: why 5%?

Comment (vamsi4845) [reply applications/style.css:355]: the text was overlapping on the view details button so i've added a margin for that

Comment (vamsi4845) [reply applications/style.css:355]: @vinit717

Comment (vamsi4845) [reply applications/style.css:355]: > @vinit717

Can You Check this pls

**Pull request merged**